### PR TITLE
 III bloks - Polymorpshim section evolutions 

### PR DESCRIPTION
**Pull request by franckbret**

### CODE REVIEW

File: en/03_blok/06_polymorphism.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
 
 ```bash
 git clone https://github.com/AnyBlok/anyblok-book-examples -b III-06-before_polymorphisme III-06-polymorphisme
 cd III-06-polymorphisme
 ```
 
+[Polymorphism][wiki_polymorphism] in AnyBlok allows you to share same database
+table with different models.
+
+It helps building elegant and comprehensive Python APi, each polymorphic model
```
Comment (petrus-v) [inline en/03_blok/06_polymorphism.md]: ``API`` ?!

**Review by petrus-v — COMMENTED** [review]
not sure about "API" otherwise LGTM

File: en/03_blok/06_polymorphism.md (hunk 1 above)
Comment (franckbret) [reply en/03_blok/06_polymorphism.md]: Yep this is a typo, it should be API

**Comment by petrus-v** [comment]
thanks, LGTM

**Pull request merged**